src/modeler/AutoRest.Swagger/Validation/Core/ValidationChangesImpact.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace AutoRest.Swagger.Validation.Core
+{
+    [Flags]
+    public enum ValidationChangesImpact
+    {
+        None = 1 << 0,
+        PotentialServiceBreakingChanges = 1 << 1,
+        NonBreakingServiceChanges = 1 << 2
```

Comment:
Definitely should not be ```NoServiceChangesRequired``` since the service itself needs to be changed so as to expose say a new end point for PATCH operation